lab1/style.css

```diff
@@ -0,0 +1,107 @@
+body {
+  padding: 182px 100px 50px 100px;
+  text-align: justify;
+  background-color: #819A9A;
+  font-size: 100%;
+  font-family: "Comic Sans MS", arial;
+
+}
+nav {
+  position: fixed;
+  width: 100%;
+  right: 100%;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  padding: 20px 0;
+  background-color: #D7ECEC;
+  border-radius: 5px;
+  text-align: center;
+
+}
+
+a {
+  text-align: center;
+}
+
+a:link {
+    color: #000000;
+}
+
+a:visited {
+    color: #000000;
+}
+
+a:hover {
+    color: #FF00FF;
+}
+
+a:active {
+    color: #0000FF;
+}
+caption {
+  margin-bottom: 15px;
+  font-size: 20px;
+  font-weight: 700;
+  font-style: italic;
+}
+
+img {
+  display: block;
+  margin: auto;
+  width: 40%;
+  filter:grayscale(100%);
+}
+
+section {
+  border: 1px solid #4C6A6A;
+  border-radius: 10px;
+  border-spacing: 5px;
+  margin-bottom: 10px;
+  background-color: #ACD0D0;
+  padding: 60px 20px;
+}
+
+h1, h2, h3 {
+  color: #002A2A;
+  font-weight:bold;
+}
+
+h2 {
+  text-indent: 10px;
+}
+
+figcaption {
+  text-align: center;
+}
+
+ul {
+  list-style-type: circle;
+  list-style-position: inside;
+}
+
+th {
+  text-align: center;
+}
+
+.float-right {
+  float: right;
+
+}
+
+.float-left {
+  float: left;
+}
+
+.wrapper {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+head, body, nav;  {
+  flex: 1 100%;
+}
+
+@media all and (min-width: 800px) {
+  body { flex: 2 0px; }
```

Comment:
Bad for many reasons:
 - Dont put it all in one line
 - `0` instead of `0px`
 - second body tag
 - You really wanna have every visible element in your website flexed?